clusters/app.ci/vault/manifests.yaml

```diff
@@ -186,7 +186,7 @@ spec:
           preStop:
             exec:
               command: ["/bin/sleep", "30"]
-      - image: vault:1.7.0
```

Comment:
This comes from an imagestream in the cluster
/hold

Reply:
From where?

Reply:
The imagestream is there. Do we have also for `subpath-proxy` ?

Reply:
It's in the very same file: https://github.com/openshift/release/blob/dfc0e6ac61aaded5799237e650dcc1ab38ba81fb/clusters/app.ci/vault/manifests.yaml#L30-L52

Subpath proxy is built from our own code so doesn't need anything

Reply:
Great.. I will remove vault from my list then :) Thanks.